### PR TITLE
nautilus: os/bluestore: be more verbose in _open_super_meta by default.

### PR DESCRIPTION
**Pull request by dvanders**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>
(cherry picked from commit 4087f82aea674df4c7b485bf804f3a9c98ae3741)

Backport tracker: https://tracker.ceph.com/issues/50549
Originally part of: https://github.com/ceph/ceph/pull/30838

### CODE REVIEW

**Review by ifed01 — APPROVED** [review]

**Comment by smithfarm** [comment]
@dvanders Please don't merge stuff into stable branches. There is a process whereby the backport PRs are put through integration testing and the merge happens via that process.

**Comment by dvanders** [comment]
@smithfarm terribly sorry. I had assumed that with a big green Merge button all necessary checks had passed.

**Comment by smithfarm** [comment]
> @smithfarm terribly sorry.

No harm done :-)

> I had assumed that with a big green Merge button all necessary checks had passed.

Those are just sanity checks. The actual testing takes place in teuthology.

**Pull request merged**
